src/components/common/Swap/components/CoinListItem/CoinListItem.tsx

```diff
@@ -2,32 +2,57 @@ import {memo} from "react";
 import {clsx} from "clsx";
 import {BN, CoinQuantity} from "fuels";
 
-import styles from './CoinListItem.module.css';
-import { useAssetImage } from "@/src/hooks/useAssetImage";
+import styles from "./CoinListItem.module.css";
+import {useAssetImage} from "@/src/hooks/useAssetImage";
 import useAssetMetadata from "@/src/hooks/useAssetMetadata";
+import SuccessIcon from "@/src/components/icons/Success/SuccessIcon";
+import {checkIfCoinVerified, VerifiedAssets} from "./checkIfCoinVerified";
+import "react-tooltip/dist/react-tooltip.css";
+import {Tooltip} from "react-tooltip";
 
 type Props = {
   assetId: string;
   balance?: CoinQuantity | undefined;
+  verifiedAssetData?: VerifiedAssets;
```

Comment:
To duplicate my comment below, we can remove this as a param, and use the hook to query this value directly here

Reply:
Done!